### PR TITLE
Made some additions in "Why we need Kubernetes and what can it do" section

### PR DESCRIPTION
**Pull request by aj11anuj**
I had made some addition of points in docs `Why we need Kubernetes and what can it do` section. I had added these points:-
- `"Scheduling flexiblity and Auto-Scaling"`
- `"Monitoring"`

Whenever we learn Kubernetes through some tutorials, they usually point out these points while explaining why kubernetes is popular, why we need it, what was the problem while managing the conatiners etc etc. Well these 2 points `Scheduling flexiblity and Auto-Scaling`, `Monitoring` are indirectly written there but i thought of explicitly mentioning these points for more clarity.

### CODE REVIEW

File: content/en/docs/concepts/overview/_index.md [hunk 1]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
   Kubernetes allows you to automatically mount a storage system of your choice, such as
   local storages, public cloud providers, and more.
-* **Automated rollouts and rollbacks**
+* **Automated rollouts and rollbacks:**
   You can describe the desired state for your deployed containers using Kubernetes,
   and it can change the actual state to the desired state at a controlled rate.
   For example, you can automate Kubernetes to create new containers for your
   deployment, remove existing containers and adopt all their resources to the new container.
-* **Automatic bin packing**
+* **Scheduling flexibility and Auto-Scalability:**
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:120]:
```suggestion
* **Scheduling flexibility and suto-scalability:**
```
For consistency.

File: content/en/docs/concepts/overview/_index.md [hunk 2]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
+  Kubernetes provides you with the flexiblity to schedule the containers
+  without writing manual scripts for getting metrices, tracing metrics,
+  different kind of environments (development, production, testing, staging etc.)
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:123]:
```suggestion
  and supports different environments such as development, production, testing, and staging.
```

File: content/en/docs/concepts/overview/_index.md [hunk 3]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
+  and helps in scaling the containers up or down, making replica of some pods in other nodes or
+  environment etc. as per your need seemlessly.
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:125]:
```suggestion
  Moreover, Kubernetes facilitates seamless scaling by automatically replicating pods across nodes or
  environments to meet your specific needs.
```

File: content/en/docs/concepts/overview/_index.md [hunk 4]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
+* **Monitoring:**
+  Kubernetes provides you with the convenience to monitor a large no. of containers,
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:127]:
```suggestion
  With Kubernetes, monitoring a large number of containers becomes hassle-free.
```

File: content/en/docs/concepts/overview/_index.md [hunk 5]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
+  and in case of failure of a container/pod it self-heals it or replaces it and notifies you about
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:128]:
```suggestion
  In the event of a container or pod failure, Kubernetes promptly self-heals or replaces them, notifying you about
```

File: content/en/docs/concepts/overview/_index.md [hunk 6]
```diff
@@ -105,27 +105,38 @@ can easily manage a canary deployment for your system.
...
+  its health check quickly by communicating with API-server and for that you don't need to write
+  "n" no. of manual scripts.
```
Comment (windsonsea) [inline content/en/docs/concepts/overview/_index.md:130]:
```suggestion
  their health status through API server communication. This eliminates the requirement of writing
  numerous manual scripts.
```

**Review by windsonsea — COMMENTED** [review]
Some text suggestions for fluency.

**Comment by aj11anuj** [comment]
Messed up somehow, will make new PR 🙂

**Pull request closed without merging**